packages/@okta/vuepress-site/docs/reference/api/oie-policy-types/index.md

```diff
@@ -2056,3 +2056,141 @@ A Mapping object links together an Application and a Policy.
   }
 }
 ```
+
+## Updates to Password Policy
+
+The Password Policy API went through a few changes as part of the Identity Engine. The section below shows only the delta from the [Password Policy API](/docs/reference/api/policy/#password-policy)
+
+Earlier, we supported setting recovery factors on the policy. While we will still continue to support it for backwards compatibility of older clients, the new password policy model does not contain recovery factors. This is now moved to the password policy rule model.
+
+### Password Policy Settings Example
+
+```json
+"settings": {
+     "password": {
+       "complexity": {
+         "minLength": 8,
+         "minLowerCase": 1,
+         "minUpperCase": 1,
+         "minNumber": 1,
+         "minSymbol": 0,
+         "excludeUsername": true
+       },
+       "age": {
+         "maxAgeDays": 0,
+         "expireWarnDays": 0,
+         "minAgeMinutes": 0,
+         "historyCount": 0
+       },
+       "lockout": {
+         "maxAttempts": 10,
+         "autoUnlockMinutes": 0,
+         "showLockoutFailures": false
+       }
+     },
+     "delegation": {
+       "options": {
+         "skipUnlock": false
+       }
+     }
+}
+```
+
+The recovery factors are now on the password policy rule configured inside the action `selfServicePasswordReset`. Below are 3 examples of how recovery factors are configured on the rule based on admin requirements.
+
+### Password Policy Rule Action Data
+
+Admin requires end users to verify with just 1 authenticator before they can recover password. Email, Sms, Voice and Okta Verify Push can be used by end users to initiate recovery. No step up authenticators are required.
```

Comment:
```suggestion
In this example, the requirement is that end users verify with just one authenticator before they can recover their password. Email, SMS, Voice and Okta Verify Push can be used by end users to initiate recovery. We know that only one authenticator is requirement because there are no step up authenticators specified.
```

Reply:
Typo in @jakubvul 's suggested change: `We know that only one authenticator is requirement` should be `We know that only one authenticator is required`.

Reply:
Fixed.